llvm/include/llvm/DebugInfo/DIContext.h

```diff
@@ -50,6 +51,7 @@ struct DILineInfo {
   // DWARF-specific.
   uint32_t Discriminator = 0;
 
+  bool IsApproximatedLine = 0;
```

Comment:
use `false` rather than `0` for a `bool` literal. Perhaps it doesn't need a default value, if it's always initialized by the ctor anyway?

Reply:
Done